### PR TITLE
Add linux-arm tarball package to release build

### LINKED ISSUE #5571: Update release build to produce additional binary archive packages
The additional binary archive packages are `tar.gz` package for `linux-arm` build, `zip` packages for `win-arm` and `win-arm64` builds.

/cc @PowerShell/powershell-maintainers

### PR DESCRIPTION
**Pull request by daxian-dbw**
## PR Summary
Fix #5571 -- Produce `linux-arm` tar.gz package in the release build.

Create `linux-arm` tarball package in our release build.
Also moved the `Linux-x64` tarball creation to Ubutntu 14.04, because `Start-PSBuild` adds additional symbolic link files when running on Red Hat Family distros.
Note that `linux-arm` can only be built on Ubuntu, so the `tar-arm` package will also be generated from the Ubuntu 14.04.

## PR Checklist

Note: Please mark anything not applicable to this PR `NA`.

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - [x] Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [NA] User facing [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - [NA] Issue filed - Issue link:
- [x] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [NA] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
    - [NA] [Add `[feature]` if the change is significant or affectes feature tests](https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/testing-guidelines.md#requesting-additional-tests-for-a-pr)

### CODE REVIEW

**Comment by anmenaga** [comment]
"linux-arm can only be built on Ubuntu" - why?

**Comment by daxian-dbw** [comment]
@anmenaga It's from build.psm1: https://github.com/PowerShell/PowerShell/blob/master/build.psm1#L288
I don't know if that has been changed now.

File: tools/releaseBuild/Images/GenericLinuxFiles/PowerShellPackage.ps1 [hunk 1]
```diff
@@ -37,6 +37,13 @@ try {
         "AppImage" { Start-PSPackage -Type AppImage @releaseTagParam }
         "tar"      { Start-PSPackage -Type tar @releaseTagParam }
     }
+
+    if ($ExtraPackage -contains "tar-arm") {
+        ## Build 'linux-arm' and create 'tar.gz' package for it.
+        ## Note that 'linux-arm' can only be built on Ubuntu environment.
+        Start-PSBuild -Runtime linux-arm -PSModuleRestore
```
Comment (TravisEz13) [inline tools/releaseBuild/Images/GenericLinuxFiles/PowerShellPackage.ps1]: Don't we need to build with at least the release tag?

**Review by TravisEz13 — CHANGES_REQUESTED** [review]
see comments

File: tools/releaseBuild/Images/GenericLinuxFiles/PowerShellPackage.ps1 (hunk 1 above)
Comment (daxian-dbw) [reply tools/releaseBuild/Images/GenericLinuxFiles/PowerShellPackage.ps1]: Good catch!

**Comment by daxian-dbw** [comment]
@TravisEz13 Your comment has been addressed. Can you please take another look? Thanks!

**Review by TravisEz13 — APPROVED** [review]

**Pull request merged**